assignment_1/2_kth_element.py

```diff
@@ -0,0 +1,101 @@
+# Implement an algorithm to find the kth to last element of a singly linked list
+# Magdalena Grodzińska
+
+# linked list implementation
+
+class Node:
+    def __init__(self, data, next_node):
+        self.data = data
+        self.next_node = next_node
+
+    def get_data(self):
```

Comment:
I didn't know that there are no private members in Python actually, thanks for pointing that out